Automattic/wp-calypso
# Redesigns New/Existing Site screen

#### Changes proposed in this Pull Request

This PR will be merged into the parent PR: #59895.

* Redesigns the New/Existing Site screen according to the designs: HpB8GsvRhJa8grFd1ikL89-fi-1048%3A1181
* This change does not change the functionality.

#### Testing instructions



* Go to `/start/do-it-for-me`
* Confirm the screen matches the design.
* Confirm that clicking on the options take you to the next screen.
* Check for desktop, tablet and mobile screen resolutions.

**Desktop**
![image](https://user-images.githubusercontent.com/5436027/151125095-a4ddfe71-8bac-4e25-991d-7cd7cf438cb8.png)

**Tablet**
![image](https://user-images.githubusercontent.com/5436027/151125255-6a94ef42-e250-4971-9d38-1f68f0a7f593.png)


**Mobile**
![image](https://user-images.githubusercontent.com/5436027/151125177-1b2a338a-0075-4dc8-afb0-a5ce0347c809.png)
